### PR TITLE
Docstr Interrogate in a seperate job

### PR DESCRIPTION
**Pull request by mohitdmak**
Hey @jakebeal , for #96 , I have moved docstr_coverage testing before the build and other tests on all platforms.

### CODE REVIEW

**Review by jakebeal — APPROVED** [review]
Looking good. I'll bet the docstr_coverage job can be used to produce a badge as a build artifact too.

**Comment by mohitdmak** [comment]
> Looking good. I'll bet the docstr_coverage job can be used to produce a badge as a build artifact too.

I have added the badge by having Docstrings check as a new workflow, is that ok as we are also going to have another workflow for linting?
Also currently a post request to "validator_url" in a few tests is not getting any response causing them to fail.

**Comment by jakebeal** [comment]
I think using the workflow badge is a reasonable solution. It would be a bonus we could get the version with the number in there, but that's a nice-to-have. Do please switch the link to be this repository rather than yours.

The test failures are because https://validator.sbolstandard.org/ is currently down due to a certificate issue; the maintainers are trying to bring it back up again right now.

File: README.md
Comment (jakebeal) [inline README.md:5]: Please change link location.

**Review by jakebeal — CHANGES_REQUESTED** [review]

**Comment by jakebeal** [comment]
Validator service is up again, so tests are working.

File: README.md
Comment (mohitdmak) [reply README.md:5]: Sure

**Comment by jakebeal** [comment]
It looks like your interrogate run is having an issue due to the relative paths. You may want to update from develop, as I ran into this issue also and put in a patch to use a path based on file location. See: https://github.com/SynBioDex/SBOL-utilities/commit/dc9b609c944172fd25f521b2db1ba4b1b1735eed#diff-21a6bd99abbaa8a90bda9afdeaf8c25f2a16b792fa62d29d23ef13083e478e51

**Comment by jakebeal** [comment]
I think this rabbit hole is getting too deep... let's just fall back to a pass/fail badge.

**Comment by jakebeal** [comment]
Please let me know when you're ready for another review.

**Comment by mohitdmak** [comment]
> Please let me know when you're ready for another review.

Yep, I just reverted to our previous pass/fail badge and fixed its link to your this repo instead of my fork, I guess it can be merged if we are finally sticking to pass/fail?

File: docstr_badge.svg
Comment (jakebeal) [inline docstr_badge.svg:1]: With the pass/fail badge plan, I believe this file isn't being referenced any more. Please delete it, and then we're good for merge.

**Review by jakebeal — CHANGES_REQUESTED** [review]

File: docstr_badge.svg
Comment (mohitdmak) [reply docstr_badge.svg]: Oh, my bad, I have got it deleted now.

**Review by jakebeal — APPROVED** [review]

**Pull request merged**
